docs/self-managed/modeler/web-modeler/configuration/database.md

```diff
@@ -0,0 +1,89 @@
+
+No certificates are needed in Web Modeler for this mode.
+
+### SSL mode "verify-full"
+
+In this mode, Web Modeler requests a certificate from the database server to verify its identity. It is not
+prone to man-in-the-middle attacks.
+
+To enable this mode, mount the root certificate with which the server certificate was signed and follow these steps:
+
+1. Provide the root certificate at this location: `myCA.crt -> ~/.postgresql/root.crt`
+1. Modify the JDBC URL: `jdbc:postgresql://[DB_HOST]:[DB_PORT]/[DB_NAME]?ssl=true`
+
+### SSL mode "verify-full" with client certificates
+
+In this mode, Web Modeler requests a certificate from the database server to verify the server's identity, and
+the server requests a certificate from the client to verify the client's identity.
+
+To enable this mode, mount the client certificates and follow these steps:
+
+1. Provide client certificates at these locations:
+   1. `myClientCertificate.pk8 -> ~/.postgresl/postgresql.pk8`
+   1. `myClientCertificate.crt -> ~/.postgresl/postgresql.crt`
+1. Provide the root certificate at this location: `myCA.crt -> ~/.postgresql/root.crt`
+1. Modify the JDBC URL: `jdbc:postgresql://[DB_HOST]:[DB_PORT]/[DB_NAME]?ssl=true`
+
+Furthermore, configure the database server to verify client certificates:
+[PostgreSQL documentation](https://www.postgresql.org/docs/current/ssl-tcp.html)
+
+## Running Web Modeler on Amazon Aurora PostgreSQL
+
+Web Modeler supports running on Amazon Aurora PostgreSQL.
+In order to connect Web Modeler with your Amazon Aurora PostgreSQL instance you have to make the following configuration adjustments:
+
+1. Modify the `SPRING_DATASOURCE_URL` environment variable: `jdbc:aws-wrapper:postgresql://[DB_HOST]:[DB_PORT]/[DB_NAME]`
+1. Add the environment variable `SPRING_DATASOURCE_DRIVER_CLASS_NAME` with the value `software.amazon.jdbc.Driver`
+
+For a full list of available driver parameters visit the [AWS JDBC Driver documentation](https://github.com/awslabs/aws-advanced-jdbc-wrapper/wiki/UsingTheJdbcDriver#aws-advanced-jdbc-driver-parameters).
+
+### AWS IAM Authentication
+
+If you'd like to use AWS Identity and Access Management (IAM) database authentication with your Amazon Aurora PostgreSQL
+instance, in addition to the adjustments described [above](#running-web-modeler-on-amazon-aurora-postgresql), follow these steps:
+
+1. Modify the `SPRING_DATASOURCE_URL` environment variable as follows: `jdbc:aws-wrapper:postgresql://[DB_HOST]:[DB_PORT]/[DB_NAME]?wrapperPlugins=iam`
+2. Remove the `SPRING_DATASOURCE_PASSWORD` environment variable
+
+Please ensure that the database user you're connecting with has the necessary permissions to authenticate via AWS IAM as
+described in the [Amazon Aurora documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html#UsingWithRDS.IAMDBAuth.DBAccounts.PostgreSQL).
```

Comment:
Do we also need to mention the `SPRING_DATASOURCE_USERNAME` here? If I remember correctly you mentioned once that it should be the IAM service account name.

Reply:
This was already covered by "Please ensure that the database user you're connecting with has the necessary permissions to authenticate via AWS IAM ..." but you're right, explicitly mentioning `SPRING_DATASOURCE_USERNAME` here makes the configuration clearer.